clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -124,28 +124,28 @@ public void setup() {
 
     @Test(expected = IllegalStateException.class)
     public void testFailIfUnreadyForSendNoProducerId() {
-        transactionManager.failIfUnreadyForSend();
+        transactionManager.failIfNotReadyForSend();
     }
 
     @Test
     public void testFailIfUnreadyForSendIdempotentProducer() {
```

Comment:
I changed a few test names to match the rename for the methods they are testing.